.github/workflows/build-and-test.yml

```diff
@@ -34,6 +34,7 @@ jobs:
         node-version: [20]
         os: [
             ubuntu-latest,
+            macos-15-intel,
```

Comment:
Let's make this `macos-latest` so it creates `aarch64` prebuilds (previously it was `macos-13` so moving to intel might have some unintended side effects).

Reply:
so my fault all along 😬  - thanks Kevin!